stdlib/LinearAlgebra/test/adjtrans.jl

```diff
@@ -504,4 +504,21 @@ using .Main.OffsetArrays
     @test_throws BoundsError s[1, 4]
 end
 
+@testset "$f(::$T{<:$E})" for T in [Adjoint, Transpose],
+                              f in [cholesky, cholesky!],
```

Comment:
this does make the tests a bit hard to read...

Reply:
What would you suggest?